stock_barcodes/wizard/stock_barcodes_read_inventory.py

```diff
@@ -65,10 +65,16 @@ def _add_inventory_line(self):
         StockInventoryLine = self.env["stock.inventory.line"]
         line = StockInventoryLine.search(self._prepare_inventory_line_domain(), limit=1)
         if line:
+            if self.product_id.tracking == "serial" and line.product_qty > 0.0:
+                self._set_messagge_info(
+                    "more_match", _("Inventory line with more than one quantity")
```

Comment:
This is the message used in core:
```suggestion
                    "more_match", _("You can only process 1.0 %(product)s of products with unique serial number.") % {"product": self.product_id.name}
```

Reply:
This message is too long

Reply:
How about?:
```suggestion
                    "more_match", _("Inventory line with more than one unit in serial tracked product")
```